<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>1080</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 25, 147);</string>
   </property>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>840</y>
      <width>301</width>
      <height>187</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_plus_move_x">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(133, 168, 137);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_move_y">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF</string>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_move_z">
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox
{
    color: rgb(50, 0, 0);
    background-color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_move_z">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF</string>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_move_y">
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox
{
    color: rgb(50, 0, 0);
    background-color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_plus_move_y">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(133, 168, 137);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_move">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(193, 203, 131);
</string>
       </property>
       <property name="text">
        <string>Move</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_minus_move_z">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(148, 168, 123);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_minus_move_x">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(148, 168, 123);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_plus_move_z">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(133, 168, 137);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_move_x">
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox
{
    color: rgb(50, 0, 0);
    background-color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-2.000000000000000</double>
       </property>
       <property name="maximum">
        <double>2.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_move_x">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_minus_move_y">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(148, 168, 123);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>580</y>
      <width>301</width>
      <height>187</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_minus_turn_y">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(148, 168, 123);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_turn_z">
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox
{
    color: rgb(50, 0, 0);
    background-color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_plus_turn_y">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(133, 168, 137);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_plus_turn_z">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(133, 168, 137);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_turn_y">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF</string>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_minus_turn_z">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(148, 168, 123);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_minus_turn_x">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(148, 168, 123);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_turn_z">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF</string>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_turn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(193, 203, 131);
</string>
       </property>
       <property name="text">
        <string>Turn</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_turn_x">
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox
{
    color: rgb(50, 0, 0);
    background-color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_plus_turn_x">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(133, 168, 137);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_turn_y">
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox
{
    color: rgb(50, 0, 0);
    background-color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_turn_x">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameForRender">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1000</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>500</y>
      <width>266</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_central">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF</string>
       </property>
       <property name="text">
        <string>Central Proection</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_parall">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF</string>
       </property>
       <property name="text">
        <string>Paralel Proection</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>220</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="4" column="2" colspan="2">
      <widget class="QLabel" name="label_size_edge">
       <property name="styleSheet">
        <string notr="true">QWidget{
font: 18px;
color: #FFFFFF
}</string>
       </property>
       <property name="text">
        <string>Size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QComboBox" name="comboBox_typ_edge">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>solid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>doted</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_type_edge">
       <property name="styleSheet">
        <string notr="true">QWidget{
font: 18px;
color: #FFFFFF
}</string>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_plus_size_edge">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(133, 168, 137);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="comboBox_typ_dots">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>circle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>square</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_minus_size_dots">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(148, 168, 123);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QLabel" name="label_edge">
       <property name="styleSheet">
        <string notr="true">QWidget{
font: 26px;
color: #FFFFFF
}</string>
       </property>
       <property name="text">
        <string>Edge</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_type_dots">
       <property name="styleSheet">
        <string notr="true">QWidget{
font: 18px;
color: #FFFFFF
}</string>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_color_edge">
       <property name="styleSheet">
        <string notr="true">QWidget{
font: 18px;
color: #FFFFFF
}</string>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_color_edge">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 162, 174);</string>
       </property>
       <property name="text">
        <string>color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_dots">
       <property name="styleSheet">
        <string notr="true">QWidget{
font: 26px;
color: #FFFFFF
}</string>
       </property>
       <property name="text">
        <string>Dots</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_minus_size_edge">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(148, 168, 123);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_color_dots">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 162, 174);</string>
       </property>
       <property name="text">
        <string>color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_color_dots">
       <property name="styleSheet">
        <string notr="true">QWidget{
font: 18px;
color: #FFFFFF
}</string>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_plus_size_dots">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(133, 168, 137);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLabel" name="label_size_dots">
       <property name="styleSheet">
        <string notr="true">QWidget{
font: 18px;
color: #FFFFFF
}</string>
       </property>
       <property name="text">
        <string>Size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>549</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
font: 26px;
color: #FFFFFF
}</string>
    </property>
    <property name="text">
     <string>Turn around</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>800</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
font: 26px;
color: #FFFFFF
}</string>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonMakeGif">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(105, 140, 196);</string>
       </property>
       <property name="text">
        <string>Gif</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButtonStopMakeGif">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(105, 140, 196);</string>
       </property>
       <property name="text">
        <string>Stop gif</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_back_color">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(122, 157, 186);</string>
       </property>
       <property name="text">
        <string>Background</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonMakeScreenshot">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(105, 140, 196);</string>
       </property>
       <property name="text">
        <string>Screenshot</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_centering">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(122, 157, 186);</string>
       </property>
       <property name="text">
        <string>Centering</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_scale">
         <property name="styleSheet">
          <string notr="true">QWidget{
font: 26px;
color: #FFFFFF
}</string>
         </property>
         <property name="text">
          <string> Scale</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_plus_scale">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(133, 168, 137);</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_minus_scale">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(148, 168, 123);</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>154</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(105, 140, 196);</string>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
